live.tsx

```diff
@@ -30,25 +28,22 @@ interface LivePageData {
 export interface LivePageOptions<Data = unknown> {
   template?: string;
   render?: (props: PageProps<LivePageData & Data>) => any;
-  loader?: (
-    req: Request,
-    ctx: HandlerContext<Data>,
-  ) => Promise<Data>;
+  loader?: (req: Request, ctx: HandlerContext<Data>) => Promise<Data>;
 }
 
 export function createLivePage<Data>(options: LivePageOptions<Data>) {
   const defaultRender = options.render;
 
   const handler: Handlers<LivePageData> = {
     async GET(req, ctx) {
-      console.log(req, ctx);
+      const { start, end, printTimings } = createServerTiming();
```

Comment:
Sugestão:
```suggestion
      const { start, end, printTimings } = createServerTiming<'fetch-page'data' | 'run-page-loader'>();
```
E tipar o `start` e `end`, só pra evitar typos.